pkg/acs/lib/acs2d/acs2d.c

```diff
@@ -197,24 +196,20 @@ void Init2DTrl (char *input, char *output) {
 	/* Start by stripping off suffix from input/output filenames */
 	if (MkOutName (input, isuffix, trlsuffix, nsuffix, trl_in, CHAR_LINE_LENGTH)) {
 		WhichError (status);
-		sprintf (MsgText, "Couldn't determine trailer filename for %s", input);
-		trlmessage (MsgText);
+		trlerror("Couldn't determine trailer filename for %s", input);
```

Comment:
Is this change from `trlmessage` to `trlerror` intentional? Same question for all similar occurrences.

Reply:
Shouldn't it be an error?

Reply:
In grand scheme of things, I don't think it matters. Looks like just a prefix change to the message put in trailer file. And `WhichError` already calls `trlerror`, so this is super confusing to begin with. I guess `trlerror` is more consistent here. Just wanted to make sure the change was intentional. Thanks!